### PR TITLE
[New Rule] Potential Office Sandbox Evasion via ZIP File

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/833

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+creation_date = "2020/01/11"
+maturity = "production"
+updated_date = "2020/01/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of suspicious zip file prepended with special chars. Sandbox Microsoft Office applications are
+allowed to write files that start with special characters which can be combined with autostart location to achieve
+sandbox evasion.
```
Comment (bm11100) [inline rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml:11]:
```suggestion
Identifies the creation of a suspicious zip file prepended with special characters. Sandboxed Microsoft Office applications on macOS are
allowed to write files that start with special characters, which can be combined with an AutoStart location to achieve
sandbox evasion.
```

File: rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential Office Sandbox Evasion"
```
Comment (bm11100) [inline rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml:17]:
```suggestion
name = "Potential Microsoft Office Sandbox Evasion"
```

File: rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml (hunk 1 above)
Comment (bm11100) [reply rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml:11]: re-lint

File: rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+risk_score = 73
+rule_id = "d22a85c6-d2ad-4cc4-bf7b-54787473669a"
+severity = "high"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Defense Evasion"]
+type = "query"
+
+query = '''
+event.category:file and not event.type:deletion and file.name:~$*.zip
+'''
+
```
Comment (bm11100) [inline rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml:32]: We could add the Mitre technique `Virtualization/Sandbox Evasion` https://attack.mitre.org/techniques/T1497/. That's likely one we don't have covered, so would be a good addition.

Comment (Samirbous) [reply rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml]: indeed but their description for T1497 is not covering process sandbox/AppContainer evasion.

File: rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+maturity = "production"
+updated_date = "2020/01/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of a suspicious zip file prepended with special characters. Sandboxed Microsoft Office applications on macOS are
+allowed to write files that start with special characters, which can be combined with an AutoStart location to achieve
+sandbox evasion.
+"""
```
Comment (threat-punter) [inline rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml:12]: This rule needs to be linted

**Review by threat-punter — APPROVED** [review]
👍

**Review by bm11100 — APPROVED** [review]

**Pull request merged**